execution-engine/src/native_modules/darknet_inference.rs

```diff
@@ -0,0 +1,202 @@
+//! A native module for ML inference on Darknet.
+//! Takes an input image, feeds it to the model and outputs a list of detected
+//! objects.
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Licensing and copyright notice
+//!
+//! See the `LICENSE_MIT.markdown` file in the Veracruz root directory for
+//! information on licensing and copyright.
+
+use crate::fs::{FileSystem, FileSystemResult};
+use crate::native_modules::common::Service;
+use darknet::{BBox, Detection, Image, Network};
+use lazy_static::*;
+use serde::Deserialize;
+use std::cmp::Ordering;
+use std::env::{current_dir, set_current_dir};
+use std::fmt::Write as _;
```

Comment:
I've never seen this. What does it do?

Reply:
It's useful when a trait requires an import but the name of the module is already taken (by std::io::Write here)